tests/integration/pilot/istioctl_test.go

```diff
@@ -76,65 +76,23 @@ var (
 9092 DestinationRule: a\..* for "a"
    Matching subsets: v1
    No Traffic Policy
-9093 DestinationRule: a\..* for "a"
-   Matching subsets: v1
-   No Traffic Policy
-81 DestinationRule: a\..* for "a"
-   Matching subsets: v1
-   No Traffic Policy
-7071 DestinationRule: a\..* for "a"
-   Matching subsets: v1
-   No Traffic Policy
-82 DestinationRule: a\..* for "a"
-   Matching subsets: v1
-   No Traffic Policy
-82 VirtualService: a\..*
+9092 VirtualService: a\..*
    when headers are end-user=jason
-82 RBAC policies: ns\[.*\]-policy\[integ-test\]-rule\[0\]
-`)
-
-	describePodAOutput = regexp.MustCompile(`Service: a\..*
-   Port: http 80/HTTP targets pod port 18080
-   Port: grpc 7070/GRPC targets pod port 17070
-   Port: tcp 9090/TCP targets pod port 19090
-   Port: tcp-server 9091/TCP targets pod port 16060
-   Port: auto-tcp 9092/UnsupportedProtocol targets pod port 19091
-   Port: auto-tcp-server 9093/UnsupportedProtocol targets pod port 16061
-   Port: auto-http 81/UnsupportedProtocol targets pod port 18081
-   Port: auto-grpc 7071/UnsupportedProtocol targets pod port 17071
-   Port: http-instance 82/HTTP targets pod port 18082
-80 DestinationRule: a\..* for "a"
-   Matching subsets: v1
-   No Traffic Policy
-80 VirtualService: a\..*
-   when headers are end-user=jason
-80 RBAC policies: ns\[.*\]-policy\[integ-test\]-rule\[0\]
-7070 DestinationRule: a\..* for "a"
-   Matching subsets: v1
-   No Traffic Policy
-7070 VirtualService: a\..*
-   when headers are end-user=jason
-7070 RBAC policies: ns\[.*\]-policy\[integ-test\]-rule\[0\]
-9090 DestinationRule: a\..* for "a"
-   Matching subsets: v1
-   No Traffic Policy
-9090 RBAC policies: ns\[.*\]-policy\[integ-test\]-rule\[0\]
-9091 DestinationRule: a\..* for "a"
-   Matching subsets: v1
-   No Traffic Policy
-9091 RBAC policies: ns\[.*\]-policy\[integ-test\]-rule\[0\]
-9092 DestinationRule: a\..* for "a"
-   Matching subsets: v1
-   No Traffic Policy
 9093 DestinationRule: a\..* for "a"
    Matching subsets: v1
    No Traffic Policy
+9093 VirtualService: a\..*
```

Comment:
this is changing since it was broken with auto allocate (https://github.com/istio/istio/issues/29422)